isis/appdata/import/img_base.tpl

```diff
@@ -0,0 +1,156 @@
+                   {% else if type == "IEEE_REAL" %} MSB
+                   {% else if type == "MAC_REAL" %} MSB
+                   {% else if type == "SUN_REAL" %} MSB
+                   {% else %} LSB_INTEGER
+                   {% endif %}
+      Base       = {% if exists("IMAGE.OFFSET.Value") %}
+                   {{ IMAGE.OFFSET.Value }}
+                   {% else %}
+                   0.0
+                   {% endif %}
+      Multiplier = {% if exists("IMAGE.SCALING_FACTOR.Value") %}
+                   {{ IMAGE.SCALING_FACTOR.Value }}
+                   {% else %}
+                   1.0
+                   {% endif %}
+    End_Group
+  End_Object
+
+  Group = Instrument
+    {% block instrument %}
+    {% endblock %}
+  End_Group
+
+  Group = Archive
+    DataSetId       = {{ DATA_SET_ID.Value }}
+    {%- if exists("IMAGE_ID") -%}
+    {%- set ProductId=IMAGE_ID.Value -%}
+    {%- else if exists("PRODUCT_ID") -%}
+    {%- set ProductId=PRODUCT_ID.Value -%}
+    {%- endif -%}
+    ImageNumber     = {{ IMAGE_NUMBER.Value }}
+    ProductId       = {{ ProductId }}
+    {% block archive %}
+    {% endblock %}
+  End_Group
+
+  Group = BandBin
+    {% block bandbin %}
+    {% set filterName=FILTER_NAME.Value %}
+    {% endblock %}
+  End_Group
+
+  Group = Kernels
+    {% block kernel %}
+    {% set spacecraft = SPACECRAFT_NAME.Value %}
+    {% set instrument = INSTRUMENT_NAME.Value %}
+    {% endblock %}
+  End_Group
+
+  {% block additional_groups %}
+  {% endblock %}
+
+End_Object
+
+Object = Translation
+  {% if exists("ptrIMAGE.Value") %}
+    {% if isArray(ptrIMAGE.Value) %}
+    DataFilePointer             = ({{ ptrIMAGE.Value.0 }}, {{ ptrIMAGE.Value.1 }})
+    {% else %}
+    DataFilePointer             = {{ ptrIMAGE.Value }}
+    {% endif %}
+  {% endif %}
+  {% if exists("RECORD_BYTES") %}
+  DataFileRecordBytes         = {{ RECORD_BYTES.Value }}
+  {% endif %}
+  {% if exists("QUBE.CORE_NULL.Value") %}
+  CoreNull             = {{ QUBE.CORE_NULL.Value }}
+  {% endif %}
+  {% if exists("QUBE.CORE_LOW_REPR_SATURATION.Value") %}
+  CoreLRS              = {{ QUBE.CORE_LOW_REPR_SATURATION.Value }}
+  {% endif %}
+  {% if exists("QUBE.CORE_LOW_INSTR_SATURATION.Value") %}
+  CoreLIS              = {{ QUBE.CORE_LOW_INSTR_SATURATION.Value }}
+  {% endif %}
+  {% if exists("QUBE.CORE_HIGH_REPR_SATURATION.Value") %}
+  CoreHRS              = {{ QUBE.CORE_HIGH_REPR_SATURATION.Value }}
+  {% endif %}
+  {% if exists("QUBE.CORE_HIGH_INSTR_SATURATION.Value") %}
+  CoreHIS              = {{ QUBE.CORE_HIGH_INSTR_SATURATION.Value }}
+  {% endif %}
+  {% if exists("QUBE.SUFFIX_BYTES.Value") %}
+  SuffixBytes          = {{ QUBE.SUFFIX_BYTES.Value }}
+  {% endif %}
+  {% if exists("QUBE.SUFFIX_ITEMS.Value") %}
+  SuffixItems          = ({{ QUBE.SUFFIX_ITEMS.Value.0 }}, {{ QUBE.SUFFIX_ITEMS.Value.1 }}, {{ QUBE.SUFFIX_ITEMS.Value.2 }})
+  {% endif %}
+  {% block translation %}
+  CubeAtts             = ""
```

Comment:
What is the need for having this in the base template? Looks like isisimport is checking for the variable regardless.